lcd/lcd_api.py

```diff
@@ -193,3 +193,7 @@ def hal_write_data(self, data):
         function.
         """
         raise NotImplementedError
+
+    def hal_sleep_us(self, data):
```

Comment:
Same thing here. Rename data to be usecs

Reply:
Done and done.